### DISCUSSION TITLE
Options to sort STIGs by how long they’ve been open for? Also, options to see total open findings AND unique findings, as in not counting the same VID twice.

### DISCUSSION THREAD

**Discussion by 404GoatFound** [NUWCDIVNPT/stig-manager#1453] [Q&A]
As said, is there any way to view this? If not, what would be the easiest approach to achieve this?

**Comment by cd-rite** [comment]
Hi @Bwall95 If you maintain history, individual reviews may show when a review was marked open (if there haven't been too many updates to that review, etc. )
![image](https://github.com/user-attachments/assets/7c661698-da9e-4bad-8a1d-451ac0fed061)

The findings report does offer an overall count of findings, and a count of individual occurences:

![image](https://github.com/user-attachments/assets/bc49c0bc-93b8-4b20-b9a1-6e6a13c53f0c)
